lldb/source/Commands/CommandObjectStats.cpp

```diff
@@ -83,13 +86,21 @@ class CommandObjectStatsDump : public CommandObjectParsed {
 
     void OptionParsingStarting(ExecutionContext *execution_context) override {
       m_all_targets = false;
+      m_summary_only = false;
     }
 
     llvm::ArrayRef<OptionDefinition> GetDefinitions() override {
       return llvm::ArrayRef(g_statistics_dump_options);
     }
 
+    StatisticsOptions GetStatisticsOptions() {
+      StatisticsOptions options;
+      options.summary_only = m_summary_only;
+      return options;
+    }
+
     bool m_all_targets = false;
+    bool m_summary_only = false;
```

Comment:
Change this ivar from:
```
bool summary_only = false;
```
to:
```
lldb_private::StatisticsOptions m_stats_options;
```
And fill things into this struct when parsing the options.